README.rst

```diff
@@ -1,13 +1,7 @@
-Digital Ocean API Python Wrapper
+DigitalOcean API Python Client
 ================================
 
-This is a fork of wiredcraft/dopy repository, this repo is created as the original repos are not being
-maintained officially.
-
-Inspired by `dop <https://github.com/ahmontero/dop>`_
-
-Forked from `dopy <https://github.com/Wiredcraft/dopy>`_
-
+Inspired by `dop <https://github.com/ahmontero/dop>`_.
```

Comment:
Why you removed this ?

Reply:
Read this README.rst --> https://github.com/Wiredcraft/dopy